documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -1,54 +1,125 @@
+> These PRs should be done in and target the `release/vX.Y.Z` or `release/miner/vX.Y.Z` branch.
+
+**Backport PR**
+- [ ] All explicitly tracked items from `Dependencies for releases` have landed
+- [ ] Backported [everything with the "backport" label](https://github.com/filecoin-project/lotus/issues?q=label%3Arelease%2Fbackport+) 
+- [ ] Removed the "backport" label from all backported PRs (no ["backport" issues](https://github.com/filecoin-project/lotus/issues?q=label%3Arelease%2Fbackport+))
+- [ ] Create a PR with title `build(vX.Y.Z): backport changes for vX.Y.Z-rcX`
+   - Link to PR: 
+- [ ] Merge PR 
+
+**Release PR**
+- [ ] Update the version string in `build/version.go` to one ending with '-rcX'. 
+    - Ensure to update the appropriate version string based on whether you are creating a node release (`NodeBuildVersion`), a miner release (`MinerBuildVersion`), or both.
+- [ ] Run `make gen && make docsgen-cli` to generate documentation
+- [ ] Changelog prep
+   - [ ] Generate changelog using the script at `scripts/mkreleaselog`
+   - [ ] Add contents of generated text to `CHANGELOG.md`
+   - [ ] Editorializing (breaking changes, new feature callouts, FIPs, actor bundles, new feature callouts)
+   - [ ] (network upgrade) Specify whether the Calibration or Mainnet upgrade epoch has been specified or not yet.
+      - Example where these weren't specified yet: [PR #12169](https://github.com/filecoin-project/lotus/pull/12169)
+- [ ] Create a PR with title `build(vX.Y.Z): release vX.Y.Z-rcX`
+   - Link to PR: 
+   - Opening a PR will trigger a CI run that will build assets, create a draft GitHub release, and attach the assets.
+- [ ] Merge the PR
+   - Opening the PR will trigger a CI run that will build assets, attach the assets to the GitHub release, public the GitHub release, and create the corresponding git tag.
+ - [ ] Update `🚢 Estimated shipping date` table
+ - [ ] Comment on this issue announcing the RC
+    - Link to issue comment:   
+
+### Testing
+> [!NOTE]
+> Link to any special steps for testing releases beyond ensuring CI is green.  Steps can be inlined here or tracked elsewhere.
+
+### Stable (non-RC) Release
+
+[//]: # (This "note" with the "diff" to apply to the RC is here to avoid the duplication in the template itself.)
+[//]: # (This is done as a visible NOTE rather than a comment to make sure it's clear what needs to be added to this section.)
+[//]: # (These comment ^^^ can be removed once the NOTE steps are completed.)
+> [!NOTE]
+> Copy/paste in the `RCX` section above and then make these changes:
+> 1. Change the version string text:
+> 
+> Update the version string in `build/version.go` to one **NOT** ending with '-rcX'
+>
+> 2. Under "Changelog prep", add
+> 
+> (network upgrade) Ensure the Mainnet upgrade epoch is specified.
+> 
+> 3. Remove this `[!Note]` and the related invisible comments.
+
+### Post-Release
+
+- [ ] Open a pull request against `master` with a merge of the `release/vX.Y.Z` branch. 
```

Comment:
Note that I'm currently skipping anything with `releases`.  I figured we can adjust once this is evaluated more as part of https://github.com/filecoin-project/lotus/issues/12072